samples/bluetooth/direct_test_mode/src/fem/Kconfig

```diff
@@ -4,56 +4,35 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-DT_COMPAT_NORDIC_NRF21540 := nordic,nrf21540-fem
-DT_COMPAT_GENERIC_FEM_2_CTRL_PIN := generic-fem-two-ctrl-pins
-
-config NRF21540_FEM_SUPPORTED
-	bool
-	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_NORDIC_NRF21540))
-
-if NRF21540_FEM_SUPPORTED
-
-config NRF21540_FEM_SPI_SUPPORTED
-	bool
-	default $(dt_nodelabel_has_prop,nrf_radio_fem,spi-if)
-
-endif # NRF21540_FEM_SPI_SUPPORTED
-
-config GENERIC_FEM_SUPPORTED
-	bool
-	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_GENERIC_FEM_2_CTRL_PIN))
-
 config FEM
 	bool "Front-end module (FEM) support"
 	default y
-	depends on NRF21540_FEM_SUPPORTED || GENERIC_FEM_SUPPORTED
+	select MPSL
+	select MPSL_FEM_ONLY
+	select MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL
```

Comment:
`MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL` cannot be enabled simply like this, as it would disallow using the sample with any other FEM then nRF21540 or even using nRF21540 without the MODE pin (this would break during build time, as `MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL` is dependent on `MPSL_FEM_NRF21540_GPIO || MPSL_FEM_NRF21540_GPIO_SPI`

I'm also not sure if it should be selected by default at all - it is rather an additional option, which can be used with nRF21540 only if the MODE pin is used or SPI is used.

Reply:
You are right. I moved it under `NRF21540_FEM` option. At this moment I would like to have it enabled automatically, it will change when we will implement automatic power control